generator/types/types.go

```diff
@@ -37,7 +38,7 @@ func (s String) Tag() string {
 
 // PrismaGoCase transforms `relevance` into `Relevance_`
 func (s String) PrismaGoCase() string {
-	return strings.Title(string(s)) + "_"
+	return cases.Title(language.Und, cases.NoLower).String(string(s)) + "_"
 }
```

Comment:
[In Go 1.18 strings.Title() is deprecated. What to use now? And how?](https://stackoverflow.com/questions/71620717/in-go-1-18-strings-title-is-deprecated-what-to-use-now-and-how)

Reply:
good question, It think the solution you chose is fine